db/controllers.js

```diff
@@ -55,3 +56,15 @@ exports.getComments = (req, res, next) => {
       next(err);
     });
 };
+
+exports.postComment = (req, res, next) => {
+  const { article_id } = req.params;
+  const { username, body } = req.body;
+  addCommentToDatabase(article_id, username, body)
+    .then((newComment) => {
```

Comment:
you could call the newComment comment instead and use the object shortcut in the .send method so the variable's name becomes the key:

 .then((comment) => {
      res.status(201).send({ comment });